src/common/components/cards/card-kebab-menu-button.scss

```diff
@@ -2,10 +2,32 @@
 // Licensed under the MIT License.
 @import '../../styles/colors.scss';
 
+.kebab-menu-callout {
+    box-shadow: 0px 6.4px 14.4px $box-shadow-132, 0px 1.2px 3.6px $box-shadow-108 !important;
+    border-radius: 4px;
+    border-width: 0px !important;
+    div.ms-Callout-main {
```

Comment:
**Important note:** this will not work once it goes through webpack. The classes that are not controlled by us need to be utilized with only the global modifier. You can look at failure-instance-panel.scss as an example.

Reply:
This also applies to the css below.